.buildkite/pipeline.yml

```diff
@@ -2,7 +2,13 @@ steps:
   - label: ":shell: Shellcheck"
     plugins:
       - shellcheck#v1.3.0:
-          files: hooks/**
+          files:
+            - hooks/**
+            - commands/**
+            - libs/**
+          options:
+            - "--color=always"
+            - "--external-sources"
```

Comment:
After correcting the typo above, this is no longer necessary

Reply:
From the testing I did it seems that this is still needed. Shellcheck won't follow the source directives if `--external-sources` is not used.

Reply:
That is really weird, I ran the pipeline without the option and shellcheck did not fail. Can you try removing it so that we can further debug the issue seeing an actual run?

Reply:
Hey @linus345! thanks again for this! are you able to try toote's suggestions so we can see the issue? thanks!

Reply:
Completely missed that this comment was made.

Sure I'll try it out tomorrow.

Reply:
Sorry for taking such a long time, pushed it without this commit now.